landing-page/components/sections/StartSection.tsx

```diff
@@ -1,22 +1,37 @@
+import { Button } from '@helpwave/common/components/Button'
 import { tw } from '@helpwave/common/twind'
-import Footer from '../Footer'
-import { HelpwaveSpinner } from '@helpwave/common/icons/HelpwaveSpinner'
 
 const StartSection = () => {
+  const githubURL = 'https://github.com/helpwave'
+  const demoURL = 'https://tasks.helpwave.de'
+
   return (
-    <div className={tw('w-screen h-screen bg-white')} id="start">
-      <HelpwaveSpinner className={tw('absolute top-[25px] left-1/2 -translate-x-1/2')} height="72" width="96" />
-      <div className={tw('relative top-[40vh] m-auto')}>
-          <div className={tw('font-space text-6xl font-bold text-center')}>helpwave</div>
+    <div className={tw('pt-48 pb-16 flex gap-32 items-center justify-center')}>
+      <div className={tw('desktop:w-1/2')}>
+        <div className={tw('font-space text-6xl font-bold')}>helpwave</div>
 
-        <div className={tw('font-sans text-2xl font-medium mt-2 text-center')}>
+        <div className={tw('font-sans text-2xl font-medium mt-2 text-end')}>
           {'empowering '}
-          <span className={tw('text-hw-primary-400')}>{'medical heroes'}</span>
-          {', united in '}
-          <span className={tw('text-hw-pool-green')}>{'technology'}</span>
+          <span className={tw('text-hw-primary-800')}>{'medical heroes'}</span>
+          {', '}
+          <br className={tw('mobile:hidden')}/>
+          {'united in '}
+          <span className={tw('text-green-600')}>{'technology'}</span>
+        </div>
+
+        <div className={tw('flex my-8 gap-8 mobile:justify-evenly')}>
+          <Button variant="tertiary" color="neutral" onClick={() => { window.open(githubURL, '_blank') }} className={tw('flex items-center gap-2')}>
+            explore
+          </Button>
+
+          <Button variant="textButton" color="neutral" onClick={() => { window.open(demoURL, '_blank') }} className={tw('flex items-center gap-2')}>
+            try our demo!
+          </Button>
```

Comment:
The two different buttons look a bit weird next to each other, but I also have no great solution, but two tertiary buttons maybe look better (can also be dismissed)

Reply:
I actually not really like the tertiary button size for this purpose. I'll dive into that later on.